dags/desktop_clients_daily.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+
+from airflow import models
+from airflow.contrib.operators.bigquery_operator import BigQueryOperator
+
+with models.DAG(
+    "clients_daily",
+    schedule_interval="0 1 * * *",
+    default_args={
+        "start_date": datetime.datetime(2019, 3, 1),
+        "email": ["dthorn@mozilla.com", "dataops+alerts@mozilla.com"],
+        "email_on_failure": True,
+        "email_on_retry": False,
+        "depends_on_past": True,
+        # If a task fails, retry it once after waiting at least 5 minutes
+        "retries": 0,
+        "retry_delay": datetime.timedelta(minutes=5),
+        "project_id": models.Variable.get("gcp_project"),
+    },
+) as dag:
+    clients_daily = BigQueryOperator(
+        task_id="clients_daily",
+        bql="sql/clients_daily_v7.sql",
+        destination_dataset_table="analysis.clients_daily_v7${{ds_nodash}}",
+        write_disposition="WRITE_TRUNCATE",
+        use_legacy_sql=False,
```

Comment:
I think we should use a custom bigquery operator so that this truncate and standard sql are the default values.

additionally we need support for `query_params`, `time_partitioning_field`, and maybe some extra validation that using truncate coincides with time partitioning and a `${{ds_nodash}}` at the end of `destination_dataset_table`